### PR TITLE
added fps counter

### PR DESCRIPTION
**Pull request by sanqro**
fps counter with BlessRNG :pray: no bugs :sweat_smile:

### CODE REVIEW

File: source/Mystic_Island-Cortex/Assets/FPSDisplay.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+using TMPro;
+
+public class FPSDisplay : MonoBehaviour
+{
+    public TextMeshProUGUI FpsText;
```
Comment (mnaray) [inline source/Mystic_Island-Cortex/Assets/FPSDisplay.cs:6]: Please use correct code conventions for variables...

File: source/Mystic_Island-Cortex/Assets/FPSDisplay.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    void Update()
+    {
+        time += Time.deltaTime;
+
+        frameCount++;
+
+        if (time >= pollingTime)
+        {
+            int frameRate = Mathf.RoundToInt(frameCount / time);
+            FpsText.text = frameRate.ToString() + " FPS";
```
Comment (mnaray) [inline source/Mystic_Island-Cortex/Assets/FPSDisplay.cs:22]: This looks like a function when it's not... First letter is never capital for variable names!!!

**Review by mnaray — CHANGES_REQUESTED** [review]
Please change that one letter to uncapitalized. The rest seems fine 👍

File: source/Mystic_Island-Cortex/Assets/FPSDisplay.cs (hunk 2 above)
Comment (sanqro) [reply source/Mystic_Island-Cortex/Assets/FPSDisplay.cs:22]: chill dude, it's a typo 😡

**Comment by mnaray** [comment]
Thanks 👍

**Pull request merged**